### ISSUE TITLE
Possible incompatibility with Virtual Box 5

### ISSUE THREAD

**Issue by pushred** [solidusjs/vagrant-solidus#13]
I recently upgraded to Virtual Box 5 by way of Docker and it looks like it may have broken the vagrant-solidus file watcher. I'm on r101572 and vagrant 1.7.4. Support for Virtual Box 5 was [added in 1.7.3][1].

The following error happens when starting a site:

```
$ vagrant site start
Installing site...
Building site...
Starting site server and watcher...
example.com is started, accessible here:
  Local URL:
    http://example.com.dev
    http://lvh.me:8085
    http://localhost:8085
  Network URL:
    http://example.com.192.168.168.104.xip.io
    http://192.168.168.104.xip.io:8085
eric:example.com eric$ /opt/vagrant/embedded/gems/gems/listen-3.0.2/lib/listen/event/queue.rb:15: warning: toplevel constant Queue referenced by Thread::Queue
/opt/vagrant/embedded/gems/gems/listen-3.0.2/lib/listen/event/loop.rb:16: warning: toplevel constant Queue referenced by Thread::Queue
/opt/vagrant/embedded/gems/gems/listen-3.0.2/lib/listen/event/loop.rb:38: warning: toplevel constant Queue referenced by Thread::Queue
/Users/eric/.vagrant.d/gems/gems/vagrant-solidus-2.0.2/lib/vagrant-solidus/site/watch.rb:46:in `wait_until_listener_or_site_is_stopped': undefined method `listen?' for #<Listen::Listener:0x0000010223ccf8> (NoMethodError)
	from /Users/eric/.vagrant.d/gems/gems/vagrant-solidus-2.0.2/lib/vagrant-solidus/site/watch.rb:27:in `block (2 levels) in execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
	from /Users/eric/.vagrant.d/gems/gems/vagrant-solidus-2.0.2/lib/vagrant-solidus/site/watch.rb:26:in `block in execute'
	from /Users/eric/.vagrant.d/gems/gems/vagrant-solidus-2.0.2/lib/vagrant-solidus/site/subcommand.rb:27:in `block in with_running_vm'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/plugin/v2/command.rb:226:in `block in with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/plugin/v2/command.rb:220:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/plugin/v2/command.rb:220:in `with_target_vms'
	from /Users/eric/.vagrant.d/gems/gems/vagrant-solidus-2.0.2/lib/vagrant-solidus/site/subcommand.rb:22:in `with_running_vm'
	from /Users/eric/.vagrant.d/gems/gems/vagrant-solidus-2.0.2/lib/vagrant-solidus/site/watch.rb:20:in `execute'
	from /Users/eric/.vagrant.d/gems/gems/vagrant-solidus-2.0.2/lib/vagrant-solidus/command.rb:43:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/cli.rb:42:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/environment.rb:301:in `cli'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/bin/vagrant:174:in `<main>'
```

As can probably be surmised from the above, the site loads, but file tasks don't seem to run.

[1]: https://github.com/mitchellh/vagrant/blob/master/CHANGELOG.md#173-july-10-2015

**Comment by joanniclaborde** [comment]
Note: the warning is fixed in Listen 3.0.3, which is not used by Vagrant yet.

**Comment by joanniclaborde** [comment]
I'm currently running Vagrant 1.7.4, VirtualBox 5.0.8 and Vagrant-Solidus 2.0.3, and all seems fine, except that warning.
